doc/howto/storage_configure.md

```diff
@@ -20,9 +20,9 @@ You can also edit the storage pool configuration by using the following command:
     lxc storage edit <pool_name>
 
 (storage-configure-volumes)=
-## Configure storage volumes
+## Configure custom storage volumes
```

Comment:
You can set some config keys on non-custom volumes such as `user.*` for example. So this section does still apply somewhat to non-custom volumes.

Reply:
With the listed commands? I only get "storage pool volume not found" when I try to change the config for a container volume.